sklego/preprocessing/pandastransformers.py

```diff
@@ -6,7 +7,7 @@
 
 
 class ColumnDropper(BaseEstimator, TransformerMixin):
-    """The `ColumnDropper` transformer allows dropping specific columns from a pandas DataFrame by name.
+    """The `ColumnDropper` transformer allows dropping specific columns from a DataFrame by name.
```

Comment:
Maybe good to mention that we use narwhals under the hood here?

Reply:
thanks for your review! Sure, I've mentioned it in the "notes" section

Reply:
I was wondering if it makes sense to mention earlier, but yeah, if it's in the notes it's probably fine.